variables.tf

```diff
@@ -328,9 +330,14 @@ variable "ingress_application_gateway_subnet_id" {
 }
 
 variable "identity_type" {
```

Comment:
Also identity_type should be `nullable = false` to make sure the condition in main.tf L158 works all the times.

Reply:
`identity_type` has a validation block, `null` won't pass the validation so I think it's ok for this case.